packages/create-app/templates/default-app/app-config.yaml.hbs

```diff
@@ -1,13 +1,14 @@
 app:
   title: Scaffolded Backstage App
-  baseUrl: http://localhost:7000
+  baseUrl: http://0.0.0.0:7000
 
 organization:
   name: Acme Corporation
 
 backend:
-  baseUrl: http://localhost:7000
+  baseUrl: http://0.0.0.0:7000
   listen:
+    host: 0.0.0.0
```

Comment:
This should be the default, so remove imo. Otherwise `::` is what we want to use to also support IPv6

Reply:
Thanks, removed